framework/src/modules/chain/components/storage/entities/block.js

```diff
@@ -188,6 +188,16 @@ class ChainBlock extends BlockEntity {
 			{}
 		);
 	}
+
+	/**
+	 * Get the last block from storage
+	 *
+	 * @param {Object} [tx] - Transaction Object
+	 * @return {Promise<BasicBlock|ExtendedBlock, NonSupportedFilterTypeError|NonSupportedOptionError>}
```

Comment:
does it return `error`? probably misleading here because it throws.
probably Error is not necessary here